src/toast/accelerator/accel.py

```diff
@@ -157,6 +163,31 @@ def accel_data_create(data):
         log = Logger.get()
         log.warning("Accelerator support not enabled, cannot create")
 
+
+@function_timer
+def accel_data_reset(data):
+    """Reset device buffers.
+
+    Using the input data array, reset to zero the corresponding device array.
+    For OpenMP target offload, this runs a small device kernel that sets the
+    memory to zero.  For jax arrays, this is a no-op, since those
+    arrays are mapped and managed elsewhere.
+
+    Args:
+        data (array):  The host array.
+    Returns:
+        None
+
+    """
+    if use_accel_omp:
+        omp_accel_reset(data)
+    elif use_accel_jax:
+        pass
```

Comment:
accel_data_reset needs a JAX implementation, I can implement that in a later PR.
otherwise, something like this should be enough: 

```python
data.data = jnp.zeroes_like(data.data)
```

Reply:
Done